OpenRA.Game/Game.cs

```diff
@@ -665,6 +666,9 @@ public static void Restart()
 
 		public static void AddChatLine(Color color, string name, string text)
 		{
+			var t = DateTime.Now;
+			var time = "{0:D4}-{1:D2}-{2:D2} {3:D2}:{4:D2}:{5:D2}".F(t.Year, t.Month, t.Day, t.Hour, t.Minute, t.Second);
```

Comment:
You can make this easier to read by using `DateTime.Now.ToString("YYYY-MM-dd HH:mm:ss")`. See https://msdn.microsoft.com/en-us/library/zdtaw1bw(v=vs.110).aspx or https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Game.cs#L59